tests/suites/test_suite_bignum_mod_raw.function

```diff
@@ -132,6 +132,7 @@ void mpi_mod_raw_cond_assign( char * input_X,
     mbedtls_mpi_mod_modulus_init( &m );
 
     TEST_EQUAL( limbs_X, limbs_Y );
+    TEST_ASSERT( limbs_X > 0 );
```

Comment:
Can you add a comment that it's for the benefit of static analyzers? Otherwise it looks like the test function is doing input validation, which may seem redundant as test inputs are under our control, so presumably correct.